core/pom.xml

```diff
@@ -100,7 +100,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
Do we need to update the `NOTICE` file and possibly some stuff under `licenses/`. This is a bit of a pain since we need to track down transitive dependencies as well. (We may want to switch over to doing this with @josephadler's tool)